test/wdio/global-script/test-cmp.tsx

```diff
@@ -1,11 +1,22 @@
-import { Component, h } from '@stencil/core';
+import { Component, h, Prop } from '@stencil/core';
 
 @Component({
   tag: 'global-script-test-cmp',
   scoped: true,
 })
-export class SiblingRoot {
+export class GlobalScriptTestCmp {
+  @Prop() renderDistCustomElementComponent = false;
+
   render() {
-    return <div>I am rendered after {Date.now() - window.__testStart}</div>;
+    return (
+      <section>
+        <div>I am rendered after {Date.now() - window.__testStart}</div>
+        {/*
+        rendering this component will fail as `<attribute-basic /`> is compiled with
+        a `dist-custom-element` output target, which will break in a lazy load environment
+      */}
+        {this.renderDistCustomElementComponent && <attribute-basic></attribute-basic>}
+      </section>
```

Comment:
should this maybe be a separate, new component so that we retain some 1:1 relationship of tests to components?